src/Template/Layout/js/app/directives/datepicker.js

```diff
@@ -13,7 +13,7 @@ import 'flatpickr/dist/flatpickr.min.css';
 
 const datepickerOptions = {
     enableTime: false,
-    dateFormat: "Y-m-d H:i",
+    dateFormat: "Z",
```

Comment:
why change format to UTC?
this also has a bad side effect: validation error on BE4 API due to a bug